clusterman/simulator/simulated_pool_manager.py

```diff
@@ -42,6 +42,8 @@ def __init__(self, cluster: str, pool: str, configs: Sequence, simulator: "simul
         self.max_capacity = self.pool_config.read_int("scaling_limits.max_capacity")
         self.max_tasks_to_kill = read_int_or_inf(self.pool_config, "scaling_limits.max_tasks_to_kill")
         self.cluster_connector = SimulatedClusterConnector(self.cluster, self.pool, self.simulator)
+        self.max_weight_to_add = self.pool_config.read_int("scaling_limits.max_weight_to_add")
```

Comment:
What are we doing with these? I don't see them being used anywhere

Reply:
I checked `git grep` and it looks like we're using it here:

`clusterman/autoscaler/pool_manager.py:            delta = min(self.max_weight_to_add, delta)`

Reply:
yeah the base pool manager expects this attribute to be set, same with `self.max_weight_to_remove`